package.json

```diff
@@ -18,7 +18,8 @@
     "webpack-dev-server": "^1.9.0"
   },
   "scripts": {
-    "start": "webpack-dev-server --progress --colors",
+    "start": "webpack-dev-server --progress --colors --port 9000",
+    "sourcemap": "webpack --dev-tool source-map",
     "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
The sourcemap npm script itself is really just for creating the sourcemap artifact so it's easy to debug the trailsy code in the browser console.

I thought it was a little challenging at first to effectively debug trailsy in the browser console.
I and others weren't initially aware how to generate a source map and debug individual js files as opposed to bundle.js

The readme file should be updated to say what the sourcemap is for and how to use it to debug the application in the browser console.

As a bigger conversation for Wednesday we could talk about:
How do we best make the project to make it easy to understand and debug the application?

Do we need to update our readme file more? Should we add github pages to the project?

I like how nice and  polished the generated github pages look.
It's easy to do and then we can just put a link to them in the readme markdown file.

If you click on the settings for the trailsy project
![settings](https://cloud.githubusercontent.com/assets/6789769/8779920/1f54a036-2ec3-11e5-81e2-1ae51122981a.png)

And then click on the button to launch the page generator
![launch](https://cloud.githubusercontent.com/assets/6789769/8780016/a3586264-2ec3-11e5-8693-8cb32285f44c.png)

Then you can end up with a starting point like this:
http://trevorackerman.github.io/trailsy/